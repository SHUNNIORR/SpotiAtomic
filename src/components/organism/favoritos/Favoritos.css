.fav-titulo-contenedor{
    display: flex;
    justify-content: center;
    margin: 20px;;
}
.titulo{
    color: white;
    font-size: 2rem;
}
.favoritos-container{
    display: flex;
    flex-direction: column;
    justify-content: center;
    margin: 0 auto;
    max-width: 1000px;
}