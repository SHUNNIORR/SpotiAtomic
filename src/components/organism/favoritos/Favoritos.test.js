import React from 'react';
import {screen, render} from '@testing-library/react';
import {shallow} from 'enzyme';
import Favoritos from './Favoritos';

describe('Favoritos', ()=>{
    it('Mostrar el tÃ­tulo',()=>{
        render(<Favoritos></Favoritos>)
        expect(screen.queryByText(/Tus favoritos/i)).toBeInTheDocument()
    })
    test('Se muestra el componente de favoritos',()=>{
        const wrapper = shallow(<Favoritos></Favoritos>);
        expect(wrapper).toMatchSnapshot();
    })
})