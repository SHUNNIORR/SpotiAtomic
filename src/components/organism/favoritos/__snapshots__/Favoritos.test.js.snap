// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favoritos Se muestra el componente de favoritos 1`] = `ShallowWrapper {}`;
